### PR TITLE
Clarify path to secrets file in getting started guides

### LINKED ISSUE #1544: Secrets getting started guide should clarify path to file
**Describe the issue**

The section "Step 2: Create a secret store Dapr component" has an example
```apiVersion: dapr.io/v1alpha1
kind: Component
metadata:
  name: my-secret-store
  namespace: default
spec:
  type: secretstores.local.file
  version: v1
  metadata:
  - name: secretsFile
    value: <PATH TO SECRETS FILE>/mysecrets.json
  - name: nestedSeparator
    value: ":"
```

it is not clear if the path ` value: <PATH TO SECRETS FILE>/mysecrets.json` is relative to the localSecretStore.yaml or to the root application folder. Leaving the example as is (without editing the path) is not working.
 
**URL of the docs**

https://docs.dapr.io/getting-started/get-started-component/

**Expected content**

A working localSecretStore.yaml example

**Screenshots**

**Additional context**

### PR DESCRIPTION
**Pull request by greenie-msft**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**

- [x] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [x] Commands include options for Linux, MacOS, and Windows within codetabs
- [x] New file and folder names are globally unique
- [x] Page references use shortcodes instead of markdown or URL links
- [x] Images use HTML style and have alternative text
- [x] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Clarify path to local secrets file in getting started guides for secret management

## Issue reference

https://github.com/dapr/docs/issues/1544

### CODE REVIEW

**Review by msfussell — APPROVED** [review]
LGTM

File: daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ spec:
     value: ":"
 ```
 
-Make sure to replace `<PATH TO SECRETS FILE>` with the path to the JSON file you just created.
+Make sure to replace `<PATH TO SECRETS FILE>` with the path (relative to the root of the application folder) to the JSON file you just created.
```
Comment (AaronCrawfis) [inline daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md]: How does this work if the component directory is not in the application directory, such as the default component folder?  Is the path determined by where `dapr run` is executed or where the component directory lives?

File: daprdocs/content/en/getting-started/get-started-component.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ spec:
     value: ":"
 ```
 
-You can see that the above file definition has a `type: secretstores.local.file` which tells Dapr to use the local file component as a secret store. The metadata fields provide component specific information needed to work with this component (in this case, the path to the secret store JSON)
+You can see that the above file definition has a `type: secretstores.local.file` which tells Dapr to use the local file component as a secret store. The metadata fields provide component specific information needed to work with this component (in this case, the path to the secret store JSON relative to where you started your app from.)
```
Comment (AaronCrawfis) [inline daprdocs/content/en/getting-started/get-started-component.md]: I'd clarify this a little more, saying it's where `dapr run` is executed. Also make sure it's aligned with the other reference.

**Review by AaronCrawfis — CHANGES_REQUESTED** [review]

File: daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md (hunk 1 above)
Comment (greenie-msft) [reply daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md]: Confirmed that it's where `dapr run` is executed.

**Review by orizohar — APPROVED** [review]

**Review by AaronCrawfis — APPROVED** [review]

**Comment by AndyPook** [comment]
could this same note should be added to the https://docs.dapr.io/reference/components-reference/supported-secret-stores/file-secret-store/ page where this component is defined/doc'd?
Thanks

**Pull request merged**